README.md

```diff
@@ -1,78 +1,83 @@
-# ZeppelinOS
+# ZeppelinOS _(zos)_
```

Comment:
I like the idea of clarifying this, but it looks weird in markdown. Maybe using a new line for `(zos)`?

Reply:
This is the recommended style of standard readme:
https://github.com/RichardLitt/standard-readme/blob/master/spec.md#title
I would prefer to follow it even in small details. My argument is that if we find a way to improve it, we should propose it upstream for everybody to benefit. If it's not worth it, we will be just discussing about subjective preferences, which is what I want to avoid following the standard.

Reply:
Makes sense!